Resources/public/js/FileUploader.js

```
@@ -107,13 +110,14 @@ function PunkAveFileUploader(options)
         success: function() {
           file.remove();
         },
-        error: function ( xhr , msg, optional ) {
-          if ( optional == 'Locked') {
+        error: function (xhr) {
+          if (xhr.status === HTTP_STATUS_CODE_LOCKED) {
             file.find(".error-image-used").show().delay(3000).fadeOut();
           }
```

I am not sure, maybe should I throw an exception in all other cases (if unexpected status code)?

---

I think it would make sense to show a different error message in these cases, otherwise we will not really see that a problem occurred